### DISCUSSION TITLE
Convert a model which contains safetensors to onnx

### DISCUSSION THREAD

**Discussion by Ar57m** [huggingface/diffusers#6982] [Q&A] [answered]
I want to convert [sd-turbo](https://huggingface.co/stabilityai/sd-turbo) to onnx, but don't know how or which script I use. it has multiple safetensors.

**Comment by asomoza** [comment, marked as answer]
There's already a converted one here: https://huggingface.co/tlwu/sd-turbo-onnxruntime.

Also there's a script [here](https://github.com/huggingface/diffusers/blob/main/scripts/convert_stable_diffusion_checkpoint_to_onnx.py) but I don't know if it works, if it doesn't work you can try opening an issue.

**Reply by Ar57m** [reply]
thanks, I converted successfully to onnx, after that I tried converting it all to ort but couldn't convert the unet to Ort. 
`Error converting /content/sdturbo/unet/model.onnx: [ONNXRuntimeError] : 1 : FAIL : inference_session.cc:806 SaveToOrtFormat Failed to save ORT format model to file: /content/converted/unet/model.ort`

**Reply by asomoza** [reply]
That script is 2 years old so probably something has changed since then on the ORT side, you can open an issue with this and see if they can update/fix it.
